ceph/ceph_admin/__init__.py

```diff
@@ -204,6 +205,8 @@ def install(self, **kwargs: Dict) -> None:
             cmd += " --nogpghceck"
 
         for node in self.cluster.get_nodes(ignore="client"):
+            if self.config.get("ibm_build"):
```

Comment:
One doubt here Vaibhav, So this if will pass even if we pass `ibm-build=False` also this if statement will work right? In that case, can we remove the =True for the ibm-build tag? Just `ibm-build` is sufficient right? Please ignore if I got it wrong